src/data/faq_de.json

```diff
@@ -44,6 +44,17 @@
                                     "FAQ-Artikel zu allgemeinen Fragen finden Sie <a href='#ramp_down_general' target='_blank'>hier</a>, FAQ-Artikel zu Konsequenzen bezüglich der App-Funktionen <a href='#ramp_down_effects' target='_blank'>hier</a> und FAQ-Artikel zu Alternativen <a href='#ramp_down_alternatives' target='_blank'>hier</a>."
                                 ]
                             },
+                            {
+                                "title": "Was kann ich jetzt tun? Was muss ich wissen?",
+                                "anchor": "ramp_down_info",
+                                "active": false,
+                                "textblock": [
+                                    "Mit dem Beginn des sogenannten Ruhemodus wird die technische Weiterentwicklung der Corona-Warn-App beendet und gleichzeitig werden die vorhandenen Funktionen des Systems reduziert. Sie können bei Bedarf Ihre importierten Zertifikate (Impfung bzw. Genesung) sichern (s.a. die Hinweise in der <a href='#ramp_down_backup' target='_blank'>FAQ zum Backup der Daten</a>). Gleiches gilt für den Export der Einträge Ihres Kontakttagebuchs.",

```

Comment:
"s.a." =>
`s.&nbsp;a.`

Reply:
Changed that in https://github.com/corona-warn-app/cwa-website/commit/044992a83eceb61d9457c3459ea61ad5e6d8c004, thanks!